service/service.go

```diff
@@ -141,10 +141,14 @@ func New(config Config) (*Service, error) {
 }
 
 // Boot starts top level service implementation.
-func (s *Service) Boot(ctx context.Context) {
+func (s *Service) Boot(ctx context.Context, uniqueApp bool) {
 	s.bootOnce.Do(func() {
+		// Boot appCatalogController only if it's unique app.
```

Comment:
This is to stop reconciling for non-unique apps. Otherwise they are adding finalizers to appCatalog CRs.